### ISSUE TITLE
Add support for telegram beta

### ISSUE THREAD

**Issue by DanGLVK** [BranchMetrics/sesame_issues#33]
Currently sesame doesn't recognize telegram beta org.telegram.messenger.beta thus the toggle inside the app doesn't work,i hope you can add support for it soon

_Enviado do meu Moto G (3rd Gen) usando [FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)_

**Comment by philwall2** [comment]
Yep, we can do this. We're planning to add Plus Messenger support, we'll add support for the Beta at the same time. Should be about 2 releases away.

**Comment by nickyreports** [comment]
Please add support to telegram X too since it is also official app.

**Comment by philwall2** [comment]
@nickyreports Telegram X doesn't support link opening. It's not possible until they do. 

Edit: they claimed months ago that they were going to support links, but it hasn't happened yet (as of a couple weeks ago when I last checked)

**Comment by philwall2** [comment]
Does anyone know how to get in touch with the Telegram X developers? We want to give this another shot but need to ask them something. Can't figure out how.

**Comment by skittles9823** [comment]
There is a telegram group where you can send bug reports and feedback, however the devs aren't there.  It's just a few volunteers which relay bugs and requests to the devs through Bitbucket I think.

https://t.me/tgandroidtests

**Comment by txyyh** [comment]
please add support to nekogram,  nekogram lite and nekogram X.
These apps are widely used in Chinese.
